test/libsolidity/syntaxTests/nameAndTypeResolution/428_bare_revert.sol

```diff
@@ -5,4 +5,4 @@ contract C {
     }
 }
 // ----
-// TypeError 2144: (81-87): No matching declaration found after variable lookup.
+// Warning 6133: (81-87): Statement has no effect.
```

Comment:
I think we would be better off with this being an error. I don't see a practical use for require without arguments so it's only an extra opportunity for people who ignore warnings to shoot themselves in the foot :)

Reply:
Maybe yes, but this is kind of unrelated. How far should we go with this?

Reply:
I'm thinking only about this particular instance and only because you're already touching the code that causes this. I'm not suggesting any big refactor :)

Reply:
This was changed because we switched from proper overload resolution to "function takes arbitrary arguments".

Reply:
Oh, so it works like that for all functions with arbitrary args? Well, ok. It's probably not worth adding a special case in the analysis just for `revert` then...